website/static/js/mailingListTreebeard.js

```diff
@@ -0,0 +1,163 @@
+'use strict';
+
+var $ = require('jquery');
+var m = require('mithril');
+var Treebeard = require('treebeard');
+var $osf = require('js/osfHelpers');
+var projectSettingsTreebeardBase = require('js/projectSettingsTreebeardBase');
+
+function expandOnLoad() {
+    var tb = this;  // jshint ignore: line
+    for (var i = 0; i < tb.treeData.children.length; i++) {
+        var parent = tb.treeData.children[i];
+        tb.updateFolder(null, parent);
+        expandChildren(tb, parent.children);
+    }
+}
+
+function expandChildren(tb, children) {
+    var openParent = false;
+    for (var i = 0; i < children.length; i++) {
+        var child = children[i];
+        var parent = children[i].parent();
+        if (child.data.kind === 'event' && child.data.event.notificationType !== 'adopt_parent') {
+            openParent = true;
+        }
+        if (child.children.length > 0) {
+            expandChildren(tb, child.children);
+        }
+    }
+    if (openParent) {
+        openAncestors(tb, children[0]);
+    }
+}
+
+function openAncestors (tb, item) {
+    var parent = item.parent();
+    if(parent && parent.id > 0) {
+        tb.updateFolder(null, parent);
+        openAncestors(tb, parent);
+    }
+}
+
+function subscribe(item, notification_type, reload) {
+    var id = item.data.node.id;
+    var url = window.contextVars.node.urls.api  + 'mailing_list/';
+    var method = notification_type === 'enabled' ? 'POST' : 'DELETE';
+    $.ajax({
+        url: url,
+        type: method,
+        dataType: 'json'
+    }).done(function(){
+        //'notify-success' is to override default class 'success' in treebeard
+        item.notify.update('Settings updated', 'notify-success', 1, 2000);
+        item.data.node.mailing_list = notification_type;
+        reload();
+    }).fail(function() {
+        item.notify.update('Could not update settings', 'notify-danger', 1, 2000);
+    });
+}
+
+function displayParentNotificationType(item){
```

Comment:
This function is unused in this file. It appears to be a [hold-over](https://github.com/CenterForOpenScience/osf.io/blob/af226bba8d0f9159151aafcdb0e69da4cd9d67d5/website/static/js/notificationsTreebeard.js#L63-L77) from `notificationsTreebeard.js`. We should remove it.